ui/src/apps/searchLogs/components/SearchHeader.tsx

```diff
@@ -133,20 +133,19 @@ export default function SearchHeader({ taskGroupID }: Props) {
     setSearchValue(e.target.value)
   }
 
-  function handleSearch(e: FormEvent<HTMLFormElement>) {
-    e.preventDefault()
+  function handleSearch() {
     createTaskGroup()
   }
 
-  function filterTreeNode(inputValue: string, treeNode: TreeNode): boolean {
-    const name = treeNode.key as string
+  function filterTreeNode(inputValue: string, legacyDataNode?: LegacyDataNode): boolean {
```

Comment:
According to the [documentation](https://ant.design/components/tree-select/#Tree-props):

```
boolean|Function(inputValue: string, treeNode: TreeNode) (should return boolean)
```

maybe you can just import TreeNode from antd?

Reply:
I see that `TreeNode` is indeed exported: https://github.com/ant-design/ant-design/blob/c7add36de674bacbd94394e105ff1bf269223f4d/components/tree-select/index.tsx#L194

Reply:
@breeswish yes, doc said that, but actually is not. this `LagacyDataNode` is added by `antd-codemod`, I dig in to `rc-tree-select`, found the `filterTreeNode` definition is using `LegacyDataNode`. https://github.com/react-component/tree-select/blob/ca1429273c21959dd4372ae71cb41c8b9d9b9d7f/src/TreeSelect.tsx#L155